### DISCUSSION TITLE
Shelly H&T Plus - un'odissea per configurarlo

### DISCUSSION THREAD

**Discussion by salvatoreboemia** [indomus/forum#2916] [HUB: Home Assistant]
Salve ho acquistato lo Shelly H&T plus, questo

<img width="300" alt="Screenshot 2024-02-16 at 20 40 19" src="https://github.com/indomus/forum/assets/20614870/02c1434e-33a0-498e-bdc5-20fdfffe656d">

Ho seguito la guida sul sito (forse non quella adatta):

- https://indomus.it/guide/integrare-sensori-shelly-ht-a-home-assistant-via-mqtt/

I passaggi finali che ho eseguito sono questi (prima di farli non c'era verso di vederlo su HA)

- Messo in modalita AP 
- Andato sul wifi dello shelly
- andato su: http://192.168.33.1
- Configurato col mio Wi-Fi (sia wifi1 che wifi 2)
- Assegnato IP Statico nell'interfaccia shelly
- Collegato al wifi del router
- Shelly sempre in modalita AP raggiungibile tramite IP Settato prima. SOLO in modalità AP
- Andato su HA, usando il componente nativo ho aggiunto tramite ip lo Shelly (sempre in modalità AP)
- Ma non trova ne sensori ne batteira ne termperatura.
- Andato di nuovo in AP mode, e configurato MQTT + WebSocket (forse quest'ultimo ha fatto il suo dovere)
- Sono riuscito a vedere i vari sensori su HA
- Configurato ed aggiunto anche da app
- Lo shelly è anche in cloud con app Shelly

Ha funzionato per un po' ma adesso il simbolo del mappamondo sul display dello shelly è scomparso c'è solo l'icona bluetooth e non si aggiorna.

Per far ricomparire l'icona devo metterlo in modalita AP premendo sul retro ed i valori si aggiornano.

Esiste una guida chiara e semplice per configurarlo? (tenendo conto di tutti i passaggi comunque fatti)

Grazie per il supporto

**Comment by snakuzzo** [comment]
Ti ho già detto sul forum...come devo fare per fartelo capire?
La guida per il modello plus non c'è da noi. Quella che hai linkato è relativa ai device di gen1...il tuo è di gen2. E poi, come già ti ho detto, l'integrazione nativa non funziona con mqtt, quindi puoi disabilitarlo.

Quanto agli altri dubbi, sulla documentazione ufficiale, avresti trovato questo -> https://www.home-assistant.io/integrations/shelly/#shelly-device-configuration-generation-2-and-3

P.S. odissea perché non ti informi bene prima di fare le cose, andando a caso

**Reply by salvatoreboemia** [reply]
Ciao. Sì mi hai scritto di chiedere nel forum. 

Anche se non trattate voi l'argomento un forum (che poi è GitHub) serve per chiedere aiuto su cose che non ci sono. 

Ho seguito più o meno tutto, ma ci sono troppe info in giro ed è difficile muoversi.

Alla fine bastava il websocket configurabile anche da app Shelly. 

Grazie per il supporto.
